Utils/Extensions/StringExtensions.swift

```diff
@@ -52,6 +52,31 @@ public extension String {
         return str as! String
     }
 
+    /**
+    Truncates a String in the middle only if it's longer than `maxLength`
+
+      "ABCDEF".truncate(4)
+      // "AB…EF"
+
+    :param: maxLength The maximum length of the String.
+
+    :returns: A truncated String
+    */
+    func truncate(#maxLength: Int) -> String {
+        if count(self) >= maxLength {
+            let index1 = advance(self.startIndex, maxLength / 2)
+            var truncatedString = self.substringToIndex(index1)
+
+            truncatedString += "..."
```

Comment:
Here are 2 screenshots for the same link: first is with 3 dots and the second with an ellipsis.
![3dots](https://cloud.githubusercontent.com/assets/3260592/8788698/01660d8e-2ef1-11e5-81ec-dcbb19a7ad0a.png)
![ellipsis](https://cloud.githubusercontent.com/assets/3260592/8788701/08cd3de0-2ef1-11e5-8304-2ea88b5a22f7.png)